File: redis/lua/create/redis_create.lua
-- ----------------------------------------------------
--	redis_create.lua
--
--				Feb/13/2015
--
-- ----------------------------------------------------
require	('text_manipulate')
require	('redis_manipulate')
-- ----------------------------------------------------
function data_prepare_proc ()
	dict_aa = {}

	dict_aa = dict_append_proc (dict_aa,"t1851","福井",29175,"1981-9-21")
	dict_aa = dict_append_proc (dict_aa,"t1852","敦賀",72154,"1981-2-5")
	dict_aa = dict_append_proc (dict_aa,"t1853","小浜",52643,"1981-7-24")
	dict_aa = dict_append_proc (dict_aa,"t1854","大野",47329,"1981-3-22")
	dict_aa = dict_append_proc (dict_aa,"t1855","勝山",82178,"1981-7-11")
	dict_aa = dict_append_proc (dict_aa,"t1856","鯖江",38567,"1981-9-21")
	dict_aa = dict_append_proc (dict_aa,"t1857","あわら",49536,"1981-1-4")
	dict_aa = dict_append_proc (dict_aa,"t1858","越前",53841,"1981-10-21")
	dict_aa = dict_append_proc (dict_aa,"t1859","坂井",91326,"1981-3-18")

	return	dict_aa
end
-- ----------------------------------------------------
print ("*** 開始 ***")

dict_aa = data_prepare_proc ()

local server_host = "host_dbase"
local server_port = 6379

local socket = require('socket')

local client = socket.connect (server_host,server_port)

for key in pairs (dict_aa)  do
	local unit_aa = dict_aa[key]
	redis_set_value_proc (client,key,unit_aa)
end

client:close()

print ("*** 終了 ***")
-- ----------------------------------------------------

File: redis/typescript/create/redis_create.ts
// ---------------------------------------------------------------
//	redis_create.ts
//
//					May/02/2020
//
// ---------------------------------------------------------------
var text_manipulate=require ("/var/www/data_base/common/node_common/text_manipulate")
// ---------------------------------------------------------------
function data_prepare_proc ()
{
	var dict_aa = new Object ()

	dict_aa = text_manipulate.dict_append_proc (dict_aa,'t1851','福井',72814,'2012-5-12')
	dict_aa = text_manipulate.dict_append_proc (dict_aa,'t1852','敦賀',18257,'2012-4-15')
	dict_aa = text_manipulate.dict_append_proc (dict_aa,'t1853','小浜',26841,'2012-12-2')
	dict_aa = text_manipulate.dict_append_proc (dict_aa,'t1854','大野',31864,'2012-3-22')
	dict_aa = text_manipulate.dict_append_proc (dict_aa,'t1855','勝山',46358,'2012-2-14')
	dict_aa = text_manipulate.dict_append_proc (dict_aa,'t1856','鯖江',63792,'2012-7-12')
	dict_aa = text_manipulate.dict_append_proc (dict_aa,'t1857','あわら',38251,'2012-3-21')
	dict_aa = text_manipulate.dict_append_proc (dict_aa,'t1858','越前',52486,'2012-7-26')
	dict_aa = text_manipulate.dict_append_proc (dict_aa,'t1859','坂井',26157,'2012-11-18')
	
	return	dict_aa
}
// ---------------------------------------------------------------
console.log ("*** 開始 ***")
//
var dict_aa = data_prepare_proc ()

var redis = require ("redis")
var client = redis.createClient (6379,'localhost')

client.on("error", function (err) {
        console.log("Redis connection error to " + client.host + ":" + client.port + " - " + err)
    })

for (var key in dict_aa)
	{
	var str_json = JSON.stringify (dict_aa[key])

	client.set(key, str_json, redis.print)
	}

client.quit()
console.log ("*** 終了 ***")

// ---------------------------------------------------------------
